TTS/encoder/utils/samplers.py

```diff
@@ -0,0 +1,102 @@
+import random
```

Comment:
Who can consider moving this to the trainers if they are also useful for other models and use cases

Reply:
I think it is not useful enough for now. If we need this in the future we can do that. But for now, the only use that I can see is in the encoders :).